scripts/deploy.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+DIR_WORK=/c4c-backend
+DIR_PERSIST=${DIR_WORK}/persist/src/main/resources
+DIR_JAR=${DIR_WORK}/service/target
+
+# Check for updated database username
+if [[ -z "${CFC_DB_USERNAME}" ]]; then
+    echo "ERROR: environment variable not set for database username"
+    exit 1
+fi
+
+# Check for updated database password
+if [[ -z "${CFC_DB_PASSWORD}" ]]; then
+    echo "ERROR: environment variable not set for database password"
+    exit 1
+fi
+
+# Check for existing db.properties file
```

Comment:
Shouldn't this check happen first? I assume this file is where you're getting the database username and password from

Reply:
This check happens before any actual logic gets executed. I think the order in which these checks happen is irrelevant, as all of them are required to make any changes to `db.properties`.

Reply:
And yah, I'll make the same changes to backend scaffold.